code/processes/rdb.q

```diff
@@ -107,7 +107,7 @@ endofday:{[date;processdata]
 	writedown[hdbdir;date];
         /-creates new changeset if this is a finspace application
         if[.finspace.enabled;
-                    changeset:.finspace.createchangeset[.finspace.database];
+		    changeset:.finspace.createchangeset[.finspace.database];
```

Comment:
indent has been (partially) changed from spaces to tabs, can you restore that to how it was before please?